api_shield_api_discovery.go

```diff
@@ -0,0 +1,164 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/goccy/go-json"
+)
+
+// APIShieldDiscoveryOperation is an operation that was discovered by API Discovery.
+type APIShieldDiscoveryOperation struct {
+	// ID represents the ID of the operation
+	ID string `json:"id"`
+	// Origin represents the API discovery engine(s) that discovered this operation
+	Origin []string `json:"origin"`
+	// State represents the state of operation in API Discovery
+	State string `json:"state"`
+	// LastUpdated timestamp of when this operation was last updated
+	LastUpdated *time.Time `json:"last_updated,omitempty"`
```

Comment:
Why `omitempty`? Does not really make a difference, but I think we always write the `LastUpdated` field

Reply:
fixed https://github.com/cloudflare/cloudflare-go/commit/80d96505236493553fee84065343ea92c44d9d58